apps/editing-toolkit/editing-toolkit-plugin/dotcom-fse/templates/class-rest-templates-controller.php

```diff
@@ -11,6 +11,28 @@
  * Based on `WP_REST_Blocks_Controller` from core
  */
 class REST_Templates_Controller extends \WP_REST_Posts_Controller {
+	/**
+	 * Override the get_items method.
+	 *
+	 * Gutenberg uses the route `/wp/v2/template_parts?wp_id=%s&context=edit` to retrieve the current post data when editing
+	 * a template or template part, so we handle that here.
+	 *
+	 * @see https://github.com/WordPress/gutenberg/blob/6c94fb7a233e849e0fff51deffba6ab2866c60f4/packages/edit-post/src/editor.js#L71
+	 *
+	 * @param WP_REST_Request $request API request for template parts.
+	 * @return WP_REST_Response
+	 */
+	public function get_items( $request ) {
+		$template_id = $request['wp_id'];
+
+		if ( $template_id ) {
+			$request->set_param( 'id', $template_id );
+			$template_part = self::get_item( $request )->data;
+			return \rest_ensure_response( array( $template_part ) );
```

Comment:
This might be a silly question, but what does the forward slash "\" do in front of a method call in PHP / WordPress? A quick search didn't surface any obvious answers 😅

Reply:
It's to indicate the function is in the global namespace, rather than within the namespace of the current file (`A8C\FSE`). Alternatively you can add `use function rest_ensure_response;` at the top of the file.